core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/upscale/StackUpscaleService.java

```diff
@@ -205,4 +194,93 @@ public int getInstanceCountToCreate(Stack stack, String instanceGroupName, int a
         int reusableInstanceCount = repair ? 0 : unusedInstanceMetadata.size();
         return stackScalabilityCondition.isScalable(stack, instanceGroupName) ? adjustment - reusableInstanceCount : 0;
     }
+
+    public List<CloudResourceStatus> upscale(AuthenticatedContext ac, UpscaleStackRequest<UpscaleStackResult> request, CloudConnector<?> connector)
+            throws QuotaExceededException {
+        CloudStack cloudStack = request.getCloudStack();
+        AdjustmentTypeWithThreshold adjustmentTypeWithThreshold = request.getAdjustmentWithThreshold();
+        try {
+            return connector.resources().upscale(ac, cloudStack, request.getResourceList(), adjustmentTypeWithThreshold);
+        } catch (QuotaExceededException quotaExceededException) {
+            return handleQuotaException(request, connector, ac, cloudStack, adjustmentTypeWithThreshold, quotaExceededException);
+        }
+    }
+
+    private List<CloudResourceStatus> handleQuotaException(UpscaleStackRequest<UpscaleStackResult> request, CloudConnector<?> connector,
+            AuthenticatedContext ac, CloudStack cloudStack, AdjustmentTypeWithThreshold adjustmentTypeWithThreshold,
+            QuotaExceededException quotaExceededException) throws QuotaExceededException {
+        List<CloudResourceStatus> resourceStatus;
```

Comment:
Is there any reason why it was declared here?

Reply:
nope :) I think it was just left like this by mistake.